dna/src/commonMain/kotlin/predicate/PredicateFunctionContractStrImpl.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.trilobiitti.trilobite.dna.predicate
+
+import com.github.trilobiitti.trilobite.dna.data.DataType
+
+class PredicateFunctionContractStrImpl: PredicateFunctionContract {
```

Comment:
Also, please enable "reformat code" option in precommit actions settings (or reformat it manually)